MSAL/test/app/ios/MSALTestAppCacheViewController.m

```diff
@@ -353,9 +353,8 @@ - (void)loadCache
             MSIDAssymetricKeyPair *keyPair = [self.keyGenerator readKeyPairForAttributes:_keyPairAttributes error:nil];
             if (keyPair)
             {
-                NSString *kid = [_popManager generateKidFromModulus:keyPair.keyModulus exponent:keyPair.keyExponent];
-                MSALTestAppAsymmetricKey *publicKey = [[MSALTestAppAsymmetricKey alloc] initWithName:self.keyPairAttributes.publicKeyIdentifier kid:kid];
-                MSALTestAppAsymmetricKey *privateKey = [[MSALTestAppAsymmetricKey alloc] initWithName:self.keyPairAttributes.privateKeyIdentifier kid:kid];
+                MSALTestAppAsymmetricKey *publicKey = [[MSALTestAppAsymmetricKey alloc] initWithName:self.keyPairAttributes.publicKeyIdentifier kid:keyPair.kid];
```

Comment:
Just for clarification: 
I just tested with 2 accounts in MSAL Test app: msidlab4 & mdidlab5 and I see they're sharing the same kid token. In a real-life scenario, customers will not share the same kid token, right? (because if all accounts are sharing the same kid token I think it's security compromised).

Reply:
Hey @hieunguyenmsft The kid will be generated for the app not the user. There is no security concern if the same kid is used for multiple users.

Reply:
Great, thank you @rohitnarula7176 for explaining it to me.

Reply:
You are welcome.